setup.py

```diff
@@ -22,7 +22,7 @@
     scripts=["scripts/s3_media_upload"],
     install_requires=[
         "boto3>=1.9.23,<2.0",
-        "botocore>=1.12.23,<2.0",
+        "botocore>=1.31.62,<2.0",
```

Comment:
For completeness, this is the minimum version claiming compatability. See https://raw.githubusercontent.com/boto/botocore/develop/CHANGELOG.rst:

```
1.31.62
=======

* enhancement:Dependencies: Add support for urllib3 2.0 for Python 3.10+
[...]
```

Reply:
As discussed in #105, yep.